content/en/security/infrastructure_vulnerability/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Infrastructure Vulnerability Management
+kind: documentation
+is_beta: true
+further_reading:
+ - link: "/security/infrastructure_vulnerability/setup"
+   tag: "Documentation"
+   text: "Setting up Infrastructure Vulnerability Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Infrastructure Vulnerability Management is not available on the US1-FED Datadog site.
+</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/GwAWNEMSqyoao2fX6" >}}
+  Infrastructure Vulnerability Management is in private beta, but you can easily request access! Use this form to submit your request today.
+{{< /callout >}}
+
+Datadog [Infrastructure Vulnerability Management][1] provides infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, and its ownership, across your organization. Once you are enrolled in the private beta, the [Container Images][2] page and banner with additional capabilities are viewable. 
```

Comment:
Datadog [Infrastructure Vulnerability Management][1] leverages infrastructure observability to detect, prioritize, manage and remediate vulnerabilities in your organization. It allows you to track the vulnerabilities to closure with the understanding of your infrastructure, its use and its ownership in your organization. Currently, the private beta offers visibility into the vulnerabilities in the [Container Images][2] on your hosts. More functionality will be added during the course of the private beta.

(Can you please replace the first screenshot to one where the banner is closed (top-right X)?)